load_balancer/load_balancer.js

```diff
@@ -25,60 +25,96 @@ app.use(express.static(__dirname + '/public'));
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.json());
 
+function checkNodeConn(node, response) {
```

Comment:
It would be nice to keep this in a separate file (technically a module) named status.js, import that object and use it here. Please write the status.js file in ES6 coding conventions and put a unit test around it. All of this work seems like worth a separate PR. So for now, remove this update to connection check and push the five node installation part in this PR.

In another PR, you can send in the status check updates with the suggested changes.

Reply:
It will be moved to the next PR and in a separate module.